tests/integration/version_migrations_test.go

```diff
@@ -101,9 +101,23 @@ func waitForPostgresReady(config *pgx.ConnConfig) bool {
 	return connected
 }
 
-func createDatabaseContainer(t *testing.T, ctx context.Context, image string, database string, username string, password string) (string, int, error) {
+func createPSQLDatabaseContainer(t *testing.T, ctx context.Context, image string, database string, username string, password string) (string, int, error) {
 	postgresPort := 5432
+	env := []string{
+		fmt.Sprintf("POSTGRES_DB=%s", database),
+		fmt.Sprintf("POSTGRES_USER=%s", username),
+		fmt.Sprintf("POSTGRES_PASSWORD=%s", password),
+	}
+	cmd := []string{
+		"postgres",
+		"-cwal_level=logical",
+		"-cmax_replication_slots=1",
+		"-cmax_wal_senders=1",
+	}
+	return createDatabaseContainer(t, ctx, image, postgresPort, env, cmd)
+}
 
+func createDatabaseContainer(t *testing.T, ctx context.Context, image string, port int, env, cmd []string) (string, int, error) {
```

Comment:
Shouldn't this just be `createContainer`?

Reply:
changed in the rebase :+1: